packages/connect/src/SmoldotProvider/SmoldotProvider.ts

```diff
@@ -267,17 +261,13 @@ export class SmoldotProvider implements ProviderInterface {
     assert(!this.#client && !this.#isConnected, 'Client is already connected');
     try {
       this.#client = await this.#smoldot.start({
-        database_content: this.#db.load(),
         chain_spec: this.#chainSpec,
         max_log_level: 3, /* no debug/trace messages */
         json_rpc_callback: (response: string) => {
             this.#handleRpcReponse(response);
         },
         database_save_callback: (database_content: string) => { 
```

Comment:
The latest version of smoldot removes this  (`database_save_calback`) as an option.  Let's get rid of this here and where we use smoldot in the extension and then upgrade smoldot.